External/FEXCore/Source/Interface/Config/Config.json.in

```diff
@@ -241,6 +241,13 @@
           "Highly likely to break any multithreaded application if disabled."
         ]
       },
+      "X87ReducedPrecision": {
+        "Type": "bool",
+        "Default": "false",
+        "Desc": [
+          "Emulates X87 floating point using 64-bit precision"
```

Comment:
Should add a comment in the description here that this will result in less-accurate emulation and maybe graphical bugs.